variants/station-g1/variant.h

```diff
@@ -27,7 +27,9 @@
 #define SX126X_DIO1 LORA_DIO1
 #define SX126X_BUSY LORA_DIO2
 #define SX126X_RESET LORA_RESET
-// #define SX126X_E22 // Not really an E22
+// Not really an E22
+// #define SX126X_DIO2_AS_RF_SWITCH
```

Comment:
The line may need to be uncommented, because station g1 do use DIO2 to control RF switch. 
The next line is correct, Station G1 does not have a TCXO so there is no need to set the TCXO voltage.